### PR TITLE
To ruby from python zh cn

### PR DESCRIPTION
**Pull request by Huibean**
Related #1074

### CODE REVIEW

**Comment by VincentZhao** [comment]
I think it should be a markdown file, instead of a html file.
Please see [the en version](https://github.com/ruby/www.ruby-lang.org/blob/master/en/documentation/ruby-from-other-languages/to-ruby-from-python/index.md).

Secondly, no need to change Gemfile.lock

**Comment by Huibean** [comment]
have put the translation in index.md

File: Gemfile.lock [hunk 1]
```diff
@@ -109,3 +109,6 @@ DEPENDENCIES
   spidr (~> 0.4)
   unicorn
   validate-website (~> 0.9)
+
+BUNDLED WITH
+   1.10.1
```
Comment (JuanitoFatas) [inline Gemfile.lock]: Could you revert this change? Because not all people are using 1.10.x. Thanks.

**Comment by JuanitoFatas** [comment]
As @VincentZhao said, please commit markdown file instead of generated html. Thank you.

**Comment by stomar** [comment]
Please provide a new PR. You must not change Gemfile.lock or _site, only submit the markdown file.

And please use a commit message that conforms to our conventions, like `Translate to-ruby-from-python (zh_cn)`.
